packages/now-cli/src/util/input/input-root-directory.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import inquirer from 'inquirer';
+
+export async function inputRootDirectory(autoConfirm: boolean) {
```

Comment:
Should we also validate the path here and ask again if it's not valid?

Reply:
Good point, I think it's better to just validate it

Reply:
Yes, good idea